src/domUpdates.js

```diff
@@ -0,0 +1,24 @@
+import Traveler from '../src/traveler.js'
+import Trip from '../src/trip.js'
+import Destination from '../src/destination.js'
+
+let domUpdates = {
+
+  traveler: {},
+  trip: {},
+  destinations: [{}],
+
+  getDestinations: () => {
+    Promise.all([
+      fetch('https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/destinations/destinations')
+    ])
+      .then(responses => Promise.all(responses.map(response => response.json())))
+      .then(responses => {
+        domUpdates.destinations = responses[0].destinations;
+      })
+      .catch(error => console.log(error))
+  },
+}
```

Comment:
something that might help with clean up is having fetch methods live in a fetch object in a fetch file (much like you've got your domUpdates!). You can then export that fetch object and call methods from it where ever you want! I think this will also help with nested Promise.alls.

Reply:
this idea is how you did it no? I might circle back on that depending on how much time i have to refactor